### PR TITLE
Add basic documentation

### PR DESCRIPTION
**Pull request by travisgroth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,17 +24,35 @@ pomerium-operator intends to be the way to automatically configure pomerium base
...
 ## Initial discussion 
 https://github.com/pomerium/pomerium/issues/273
+
 https://github.com/pomerium/pomerium/issues/425
 
 # Installing
-TBD helm chart integration
+The pomerium operator should be installed with the pomerium helm chart at https://helm.pomerium.io
+
+It may be run from outside the cluster for development or testing by using your current kubernetes context.
```
Comment (desimone) [inline README.md]: Can you provide a little more detail as to what "your current kubernetes context" means here? https://kubernetes.io/docs/reference/kubectl/cheatsheet/#kubectl-context-and-configuration ?

File: README.md [hunk 2]
```diff
@@ -23,17 +24,35 @@ pomerium-operator intends to be the way to automatically configure pomerium base
...
 # Building
 pomerium-operator utilizes [go-task](https://taskfile.dev/#/) for development related tasks.  
 
 `task build`
 
-# Spec
+# Using
+
+Due to current capabilities, the pomerium-operator is most useful when utilizing [forward auth](https://www.pomerium.io/configuration/#forward-auth).  At this time, you must provide the appropriate annotations
+for your ingress controller to have pomerium protect your endpoint.  Examples can be found in the pomerium documentation.
```
Comment (desimone) [inline README.md]: Maybe link to the forward auth recipe and doc? 
https://www.pomerium.io/configuration/#forward-auth
https://www.pomerium.io/recipes/kubernetes.html

File: README.md [hunk 3]
```diff
@@ -48,16 +67,62 @@ Policy is set by annotation, as are typical Ingress Controller semantics.
...
-
-3. Get "table stakes" Ingress features into pomerium.  Target model is Inverted Double Ingress or Simple Ingress.  We need cert handling up to snuff, but load balancing and path based routing can be offloaded to a next-hop ingress controller or kube-proxy via Service.  CRD maps which "next-hop" service to use for the IDI model from the ingress class.
-
-4.  Introduce backend load balancing via Endpoint discovery to allow for skipping a second ingress for most configurations.
-
-5.  Allow non-Ingress/Service based policy via CRD.  Helm chart does conversion on the backend.
-
-6.  Pomerium deployment itself is managed by CRD.  The helm chart becomes a wrapper to this CRD.  Move the templating and resource generation logic into pomerium-operator.
\ No newline at end of file
+## Example
```
Comment (desimone) [inline README.md]: This example is really helpful 👍

File: README.md (hunk 1 above)
Comment (travisgroth) [reply README.md]: Hopefully the phrasing makes more sense now.

File: README.md (hunk 2 above)
Comment (travisgroth) [reply README.md]: 👍

**Pull request merged**